apps/schools/models/assessments.py

```diff
@@ -162,8 +163,8 @@ class AnswerStudent(models.Model):
             blank=True, null=True)
     last_modified_by = models.ForeignKey(User, blank=True, null=True,
             related_name='last_modified_answer_student')
-    flexi_data = models.CharField(max_length=30, blank=True, null=True)
-    active = models.IntegerField(blank=True, null=True, default=2)
+    flexi_data = models.CharField(max_length=30, blank=True, null=True) #Shivangi, is this needed?
```

Comment:
This is not really necessary now i guess. This was a feature added to EMS by mahiti and us, which was not really useful.